spanner-jdbc-cleanup - Step 05 - Add missing test cases

olavloite opened the pull request:
Adds miscellaneous unit tests that had not yet been merged in.
Two test cases in the `JdbcAbortedTransactionTest` will fail until #5952 has been merged.

kolea2 commented:
Will review after rebase :)

olavloite commented:
> Will review after rebase :)

Done rebasing, and I cherry-picked the fix for #5951 into this PR in order to prevent the `JdbcAbortedTransactionTest` to fail.

kolea2 commented on google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/ConnectionStatementWithNoParametersTest.java:
```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2018 Google LLC
```
2019

kolea2 commented on google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/DdlClientTest.java:
```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 Google LLC
```
2019

kolea2 commented on google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/ReadOnlyStalenessTest.java:
```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2018 Google LLC
```
2019

kolea2 commented on google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/SpannerJdbcExceptionMatcher.java:
```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Google LLC
```
2019

kolea2 commented on google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/StatementTimeoutTest.java:
```diff
@@ -0,0 +1,961 @@
+/*
+ * Copyright 2018 Google LLC
```
2019

kolea2 commented on google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/stresstest/DatabaseClientStressTest.java:
```diff
@@ -0,0 +1,548 @@
+/*
```
This is great, but I don't think this is the appropriate place to put this type of test alongside other JUnit tests. For example, we have a different module for our conformance tests - not sure if we should think about a new module for stress testing.

That said, can we move this to a future PR after we think about it some more?

olavloite commented:
Removed stress tests and updated copyright years.

Note that the stress tests were not executed alongside with the unit tests, as they were added to a test group that was excluded from normal test execution. If we want to add these back to the library, we should also add this exclusion again to prevent test execution from taking too much time.
